cfgov/jinja2/v1/_includes/macros/category-slug.html

```diff
@@ -0,0 +1,57 @@
+
+{# ==========================================================================
+
+   category_slug.render()
+
+   ==========================================================================
+
+   Description:
+
+   Render a category slug when given:
+
+   category:           A string.
+
+   href (optional):    If present creates a link with
+                       a path to which the category filter applies.
+                       For example, if the slug is used on a blog article
+                       then path should be '/blog/'.
+                       Remember to leverage vars.path instead of
+                       using the literal string '/blog/'.
+                       Path is used to create the filtered URL:
+                       {{ href }}?filter_category={{ category }}
+
+   classes (optional): Space separated list of class names.
+
+   use_blog_category:  Whether to use the blog category filter or not.
+                       Defaults to false.
+
+   ========================================================================== #}
+
+{% macro render(category, href, classes, use_blog_category=false) %}
+    {% import 'macros/category-icon.html' as category_icon %}
+
+    {% if href %}
+        {% if use_blog_category %}
+            {% set filter_path = href + '?filter_blog_category=' + category | urlencode | replace('%20', '+') %}
```

Comment:
This is just to support the existing calls to the category_slug.render() method. I'll add a TODO. And oops, change the name to `href` so it's actually used! Please check in https://github.com/cfpb/cfgov-refresh/commit/2b45deeaa2c6c28905bd01a48e31c23558918cb1